static/nodejs_server/line_tile.ts

```diff
@@ -97,15 +97,32 @@ export async function getLineTileResult(
     // If it is a single line chart, add highlight information.
     if (chartData.dataGroup && chartData.dataGroup.length === 1) {
       const dataPoints = _.cloneDeep(chartData.dataGroup[0].value);
-      if (!_.isEmpty(dataPoints)) {
+      const highlightDate = statVarSpec[0].date;
+      let highlightPoint = null;
+      if (!_.isEmpty(dataPoints) && highlightDate) {
+        highlightPoint = dataPoints.find(
+          (point) => point.label === highlightDate
+        );
```

Comment:
TODO: consider if we want to allow for monthly stats if year is asked

Reply:
added